sopel/coretasks.py

```diff
@@ -1313,8 +1308,8 @@ def recv_who(bot, trigger):
 @plugin.priority('medium')
 def end_who(bot, trigger):
     """Handle the end of a response to a ``WHO`` command (if needed)."""
-    if 'WHOX' in bot.isupport:
-        who_reqs.pop(trigger.args[1], None)
+    # Currently, there is nothing to handle at the end of WHO reponse.
+    pass
```

Comment:
Might as well delete the whole function. No reason to keep an empty callable around eating up resources (especially since it's unthreaded) unless there's some action to perform.

We won't lose any magic incantations to the sands of time; the decorators on this are pure boilerplate.